### ISSUE TITLE
Screenshot carousel is not visible when clicking thumbnail in mobile view from the metadata pop-in menu

### ISSUE THREAD

**Issue by szakarias** [dart-lang/pub-dev#7390]

**Comment by szakarias** [comment]
It seems it shows up somewhere because the screen locks after clicking the thumbnail.

**Comment by isoos** [comment]
I think I have narrowed the root cause: we lookup the carousel by `document.getElementById('-screenshot-carousel')`, but on the package page we have two elements with that ide (because we render the infobox twice). Couple of options to fix:
- render infobox only once (requires style updates)
- render the carousel outside of the infobox (seems like the best option)
- update the carousel logic to know which one it should use (increases client complexity/fragility)
